x/magpie/internal/keeper/querier.go

```diff
@@ -5,89 +5,46 @@ import (
 
 	"github.com/cosmos/cosmos-sdk/codec"
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	"github.com/kwunyeung/desmos/x/magpie/internal/types"
+	"github.com/desmos-labs/desmos/x/magpie/internal/types"
 	abci "github.com/tendermint/tendermint/abci/types"
 )
 
 // query endpoints supported by the magpie Querier
 const (
-	QueryPost = "post"
-	QueryLike = "like"
+	QuerySession = "testSession"
```

Comment:
It looks this would give error when query `sessions/` endpoint.

Reply:
Fixed, thanks